release-notes/4.7.21.md

```diff
@@ -0,0 +1,511 @@
+
+  This resolves a problem where the Total Opens count was only querying
+  `DISTINCT` open results.
+
+- **[CRM-20789](https://issues.civicrm.org/jira/browse/CRM-20789) Unsubscribing
+  from an AB test does not work in 4.7.21
+  ([10583](https://github.com/civicrm/civicrm-core/pull/10583))**
+
+### CiviMember
+
+- **[CRM-20675](https://issues.civicrm.org/jira/browse/CRM-20675) Membership
+  status update creates renewal activity
+  ([10598](https://github.com/civicrm/civicrm-core/pull/10598))**
+
+### Backdrop Integration
+
+- **[CRM-20612](https://issues.civicrm.org/jira/browse/CRM-20612) Error:
+  undefined function config() when installing on Backdrop
+  ([10389](https://github.com/civicrm/civicrm-core/pull/10389))**
+
+### Drupal Integration
+
+- **[CRM-20698](https://issues.civicrm.org/jira/browse/CRM-20698) Make
+  "disabled" table rows appear greyed-out
+  ([10481](https://github.com/civicrm/civicrm-core/pull/10481))**
+
+  CSS in the Seven theme conflicts with CiviCRM's CSS, preventing rows of
+  disabled records from having grey text.
+
+- **Improve Views checkbox value handling
+  ([448](https://github.com/civicrm/civicrm-drupal/pull/448) and
+  [449](https://github.com/civicrm/civicrm-drupal/pull/449))**
+
+- **[CRM-19303](https://issues.civicrm.org/jira/browse/CRM-19303) CKEditor
+  configuration can't be edited on a Drupal multisite installation
+  ([10577](https://github.com/civicrm/civicrm-core/pull/10577) and
+  [10511](https://github.com/civicrm/civicrm-core/pull/10511))**
+
+## <a name="misc"></a>Miscellany
+
+- **[CRM-20744](https://issues.civicrm.org/jira/browse/CRM-20744) Use
+  bower_compoents images for datatable rather than packages
+  ([10520](https://github.com/civicrm/civicrm-core/pull/10520))**
+
+- **[CRM-20561](https://issues.civicrm.org/jira/browse/CRM-20561) Load
+  Net_SMTP, Auth_SASL, Net_Socket via Composer
+  ([10384](https://github.com/civicrm/civicrm-core/pull/10384),
+  [4](https://github.com/civicrm/civicrm-core/pull/4),
+  [10385](https://github.com/civicrm/civicrm-core/pull/10385),
+  [185](https://github.com/civicrm/civicrm-packages/pull/185), and
+  [186](https://github.com/civicrm/civicrm-packages/pull/186))**
+
+- **[CRM-20687](https://issues.civicrm.org/jira/browse/CRM-20687) Issue numbers
+  used as variables
+  ([10473](https://github.com/civicrm/civicrm-core/pull/10473))**
+
+- **(NFC) re add in animated-overlay.gif is no longer provided by jquery-ui
+  ([10523](https://github.com/civicrm/civicrm-core/pull/10523))**
+
+- **(NFC) Fix clearing messages to ensure test accuracy
+  ([10525](https://github.com/civicrm/civicrm-core/pull/10525))**
+
+- **(NFC) Attribution Chirojeugd Vlaanderen
+  ([10519](https://github.com/civicrm/civicrm-core/pull/10519))**
+
+- **[CRM-8597](https://issues.civicrm.org/jira/browse/CRM-8597) PHP strict
+  warning: Only variables should be assigned by reference.
+  ([3](https://github.com/civicrm/civicrm-core/pull/3))**
```

Comment:
Not a huge deal... but this looked odd (very low PR#) and seems to be incorrect.

Seamus had submitted a PR, and I proposed a subordinate PR (to update his). That PR was `https://github.com/seamuslee001/civicrm-core/pull/3`. When approved, Github autogenerated a [merge-commit](https://github.com/totten/civicrm-core/commit/e737bf09dd174920d02e4774872ddfac0c3539c5) with the ambiguous phrase `#3`, and this got mixed up with `https://github.com/civicrm/civicrm-core/pull/3`.